deployment/helm/templates/deployment.yaml

```diff
@@ -102,6 +100,8 @@ spec:
             mountPath: /secrets/app
           - name: identity-secrets
             mountPath: /secrets/identity
+          - name: sigstore-tuf-cache
+            mountPath: /tmp/minder-cache
```

Comment:
Might not be relevant, but do we need to be careful of tmp being world readable/writable?